packages/node_modules/@node-red/nodes/core/logic/10-switch.js

```diff
@@ -72,6 +71,9 @@ module.exports = function(RED) {
             var index = parts.index;
             return ((min <= index) && (index <= max));
         },
+        'hask': function(a, b) {
+            if (typeof b !== "object" ) {  return a.hasOwnProperty(b+""); }
```

Comment:
Would prefer to have explicit return in all cases, rather than just falling out of the function.
```suggestion
            return (typeof b !== "object" )  &&  a.hasOwnProperty(b+"");
```